vllm/model_executor/models/clip.py

```diff
@@ -0,0 +1,194 @@
+"""Minimal implementation of CLIPVisionModel intended to be only used 
```

Comment:
It would be nice to unit test the model in isolation to ensure its consistency with HF.

Reply:
Sure. The vision language model test should already serve as an end-to-end integration test for `CLIPVisionModel` since it's only intended to be used there, but I can add separate unit tests for this model itself.